ambari-server/src/main/java/org/apache/ambari/server/agent/CommandRepository.java

```diff
@@ -234,6 +234,13 @@ public void setPreInstalled(String isPreInstalled) {
       this.m_isPreInstalled = isPreInstalled.equalsIgnoreCase("true");
     }
 
+    public Boolean getPreInstalled() {
+      return m_isPreInstalled;
+    }
+
+    public boolean isScoped() {
```

Comment:
why this getter is needed? it is not used anywhere and the only one role he have - be serialized to json